opta/registry.py

```diff
@@ -99,18 +102,40 @@ def _make_module_docs(vanilla_text: str, module_dict: Dict[Any, Any]) -> str:
     return result
 
 
-def _make_module_registry_dict(directory: str) -> Dict[Any, Any]:
+def _get_all_module_info(directory: str, cloud: str) -> list:
+    rtn_list = []
+    all_yaml_files = glob.glob(directory + "/**/*.yaml", recursive=True)
+    for a_yaml_path in all_yaml_files:
+        try:
+            module_dict = yaml.load(open(a_yaml_path))
+            if not module_dict:
+                continue
+        except:  # nosec # noqa
+            continue
+        if "clouds" not in module_dict:
```

Comment:
when is this the case?

Reply:
If there is some other yaml file in the directory (say a test yaml file) then it will be ignored due to this logic since it wont have the clouds key.